.github/workflows/godwoken-tests.yml

```diff
@@ -13,3 +13,6 @@ jobs:
       extra_github_env: |
         GODWOKEN_TESTS_REPO=${{ github.repository }}
         GODWOKEN_TESTS_REF=${{ github.ref }}
+        MANUAL_BUILD_WEB3=true
```

Comment:
when can we remove this? after kicker update with the new web3 version?

Reply:
Yes..

Reply:
if we merge this pr, will it affect the interaction test of other CIs, such as alphabet / testnet?

Reply:
Oh, yes, CI will fail on alphanet / testnet. That is a problem.
What about changing to compatible assertion?

```
      expect(error.message + error.error.toString()).to.include(keyWord);
```

Reply:
great, I think this is a better solution!

Reply:
Updated in e69db8d.
And drop the updates for godwoken-tests.